lib/elements/prompt.js

```diff
@@ -12,7 +12,7 @@ const color = require('kleur');
  * @param {Stream} [opts.stdout] The Writable stream to write readline data to
  */
 class Prompt extends EventEmitter {
-  constructor(opts={}) {
+  constructor(opts = {}) {
```

Comment:
Looks like there is some auto-formatting going on. Should we introduce something like prettier to avoid this in the future?

Reply:
Would be a good idea ... this would also allow to unify the code style

Reply:
Fixed!